etc/gohan.yaml

```diff
@@ -53,6 +53,10 @@ keystone:
     user_name: "admin"
     tenant_name: "admin"
     password: "gohan"
+    # Whether should use cache for storing and fetching authorizations
+    use_auth_cache: true
```

Comment:
Default is set as false in the following code, so we should set false in this example config file.

Reply:
done

Reply:
done? Maybe you forgot to update here.

Reply:
oops, lost this change during branch switching. Added again ;)